### PR TITLE
work on #2425 : added an uninstall.sh script and network configuration in install.sh

### PR DESCRIPTION
**Pull request by fviale**
- The uninstall script removes all versions of the scheduler installed.
- install.sh now display the network interfaces information and let the user select one.
- improved the install.sh script to give a more next/next/next feel

### CODE REVIEW

File: tools/install.sh [hunk 1]
```diff
@@ -122,15 +128,24 @@ read -e -p "Group of the user starting the ProActive service: " -i "proactive" G
...
     else
-        useradd $USER -d $PA_ROOT -g $GROUP
+        exit 1
     fi
-    passwd $USER;
+
 fi
 
+chown $USER:$GROUP $PA_ROOT
 chown -R $USER:$GROUP $PA_ROOT/$CURRENT_PA_DIR
```
Comment (paraita) [inline tools/install.sh]: is `CURRENT_PA_DIR` a typo ? You declare `CURRENT_PADIR` at the beginning of the file.

**Comment by paraita** [comment]
the rest looks fine to me ! :+1:

File: tools/install.sh (hunk 1 above)
Comment (fviale) [inline tools/install.sh]: Well spotted paraita !

Indeed it was a mistake.

**Comment by paraita** [comment]
:+1:

**Pull request merged**